clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/mapreduce/datawriter/reduce/VeniceReducer.java

```diff
@@ -68,9 +69,12 @@ public void reduce(
     } else {
       dataWriterTaskTracker = getDataWriterTaskTracker();
     }
+
+    // TODO: Figure out how to make logical timestamp work for MR
```

Comment:
Do we plan to make it work? I don't think we need to. We should be removing the MR mode soon